commune_app/tests/test_commune.py

```diff
@@ -8,28 +8,22 @@
 NAME1 = "test_commune1"
 DESCRIPTION1 = "description1"
 
-USERNAME_IN_DB = "testUser123"
-
-
-@pytest.fixture
-def commune0():
-    return Commune(name=NAME, description=DESCRIPTION)
-
 
 @pytest.fixture
 @pytest.mark.django_db()
-def my_commune0(commune0):
+def commune0():
+    commune0 = Commune(name=NAME, description=DESCRIPTION)
     commune0.save()
     return commune0
 
 
 @pytest.mark.django_db()
 class TestComuune:
 
-    def test_my_commune0(self, my_commune0):
-        assert Commune.objects.get(id=my_commune0.id)
-        assert NAME == my_commune0.name
-        assert DESCRIPTION == my_commune0.description
+    def test_my_commune0(self, commune0):
```

Comment:
please add tests of validation of the parameters:
 - valid/invalid length
 - valid/invalid characters/numbers/signs
 - mandatory fields